atlassian-terraform/atlassian.auto.tfvars

```diff
@@ -0,0 +1,62 @@
+# Sumo Logic - Atlassian Terraform
+# Configure Jira Cloud, Jira Server, OpsGenie, BitBucket credentials and parameters.
+
+
+# Jira Cloud
+jira_cloud_url      = "https://arunpatyal.atlassian.net"
+jira_cloud_user     = "arunpatyal0809@gmail.com"
+jira_cloud_password = "AVQYnQwD1IdI2GylyUQx99B6" # Password or API Key, To generate API Key: https://confluence.atlassian.com/cloud/api-tokens-938839638.html
```

Comment:
Wow, need to change this immediately

Reply:
replace with things like "me@example.com", "mycloudpassword"

Reply:
Wrong file was uploaded. Correct file with placeholders has been uploaded and all the credentials are revoked.